testapi.pm

```diff
@@ -2127,13 +2127,16 @@ sub diag (@args) { bmwqemu::diag(@args) }
 Return VM's host IP.
 In a kvm instance you reach the VM's host under default 10.0.2.2
 
+Optional C<$force_worker_hostname> argument will force using actual worker IP
+even on KVM guests.
```

Comment:
```suggestion
Optional C<$force_worker_hostname> argument will force using actual worker IP/hostname
even on KVM guests.
```

Maybe this could also state *why* this is possibly useful.